README.md

```diff
@@ -1,2 +1,81 @@
 # dockstore-load-tester
 Dockstore Load/Performance/Stress Tester
+
+## Running the tests
```

Comment:
Any dependencies different from Dockstore (i.e. same maven, Java versions. etc.).

Reply:
The scala version is different, 2.12.7 here, required by Gatling, vs. 2.11.11 in Dockstore. My first thought had been to make this a Maven submodule of the Dockstore project, but partially because of that conflict I moved into its own repo.